cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -1485,6 +1485,12 @@ A:
             &src,
         )
         .expect_error_at_line_col(&src, YaccGrammarErrorKind::PrematureEnd, 1, 17);
+        let src = "// 🦀".to_string();
```

Comment:
While this is a Yacc test rather than a Lex one, It appears to be a nice way to produce this error that doesn't involve invalid code gen.

I added it here, because i'm not certain that accepting multibyte whitespace is actually intended, or something we just accidentally accept.
If that is the case, (I don't believe lex accepts comments), but if we added them we could reproduce PrematureEnd with a comment like this.